### DISCUSSION TITLE
Daisyui Tabs

### DISCUSSION THREAD

**Discussion by muadhaddabah** [saadeghi/daisyui#577] [Q&A] [answered]
How come the tabs don't switch or hide content on the inactive tabs?

**Comment by bbuchanan** [comment]
Came here to ask this very same thing. Maybe we have to do this manually in our code?

**Reply by adrianowy** [reply]
I'm thinking to combine part of library https://www.radix-ui.com with daisyui

**Comment by saadeghi** [comment, marked as answer]
Tabs are the visual representation. They can be links to other pages or do anything.  
If you want them to show/hide a block of content, you need to do it using JS (based on the JS framework you're using).  
There are many ways to do that but here's an example: https://svelte.dev/repl/28d9fef77b034460b6e072478c5f54f8?version=3.46.4

**Comment by gotjoshua** [comment]
I managed to get an almost purely css solution for tabs.
Only uses js for adding and removing the tab-active class.
Otherwise it is using checkbox and peer:checked strategy

code:
https://gitlab.com/onezoomin/ztax/cdn.zt.ax/-/blob/trunk/apps/color-schemes/components/tab-box.ts

live (requires a free fission account):
https://f.cdn.zt.ax/apps/color-schemes/

**Comment by Mashiane** [comment]
![Tabs](https://user-images.githubusercontent.com/5799306/189716290-08d58bf1-5e7b-4c9a-8cbf-9d49d32649c5.jpg)

![DaisyTabs](https://user-images.githubusercontent.com/5799306/189716783-0994e5d5-5cfd-4ba7-818b-6fce6558edbb.gif)

Just in case someone needs a quick outline of how these can work easily. One has then to use JS to switch between the active / non-active tabs. This might help so that one does not have to have to look for a solution.

Hope it helps someone.
